utils/switcher/throttle-server.js

```diff
@@ -95,8 +95,8 @@ const writeSlowly = (req, res, data) => {
 // maps file extention to MIME type
 // we only care about these two types
 const map = {
-  '.ts': 'application/octet-stream',
-  '.m3u8': 'application/x-mpegurl'
+  '.ts': 'application/vnd.apple.mpegurl',
```

Comment:
I think these two are reversed. And did these have to be changed to get Safari to work?

Reply:
Yep, just noticed that 👍